src/components/backend-ai-resource-preset-list.ts

```diff
@@ -551,7 +551,10 @@ class BackendAiResourcePresetList extends BackendAIPage {
   }
 
   _createPreset() {
-    const wrapper = v => v !== undefined && v.includes('Unlimited') ? 'Infinity' : v;
+    const wrapper = (v) => {
+      v = v.toString();
+      return v !== undefined && v.includes('Unlimited') ? 'Infinity' : v;
```

Comment:
In JavaScript, comparison of undefined  `!== undefined` will generate strange behavior through browsers. The correct comparison is `typeof(v) !== 'undefined'` since JavaScript is prototype-based language.

Reply:
Updated!